01-software_for_modeling.Rmd

```diff
@@ -59,7 +59,7 @@ Examples of creating a pit of success (discussed in more details later)
     -   **Classification:** ordered or unordered qualitative values.
 -   **Quantitative** data: numbers.
 -   **Qualitative** (nominal) data: non-numbers.
-    -   *Qualitative data still might be coded as numbers, e.g. one-hot encoding*
+    -   *Qualitative data still might be coded as numbers, e.g. one-hot encoding a.k.a dummy variable encoding*
```

Comment:
editor-nerd-moment - can you change it to one-hot encoding `or` dummy variable (or one-hot/dummy-variable encoding)? having both e.g. and a.k.a. in a sentence seems odd!

Reply:
done!